src/server/config.js

```diff
@@ -56,6 +56,9 @@ export default function (baseConfig, configDir) {
     module: {
       ...config.module,
       // We need to use our and custom loaders.
+      preLoaders: [
```

Comment:
Yep, this won't work, if we do it ^ this way and `customConfig.module` contains `loader` it will override `loader` in config.module.